django/contrib/gis/forms/widgets.py

```diff
@@ -78,26 +78,29 @@ class OpenLayersWidget(BaseGeometryWidget):
     template_name = 'gis/openlayers.html'
 
     class Media:
+        css = {
+            'all': (
+                '//cdnjs.cloudflare.com/ajax/libs/ol3/3.15.1/ol.css',
+                'gis/css/ol3.css',
+            )
+        }
         js = (
-            'http://openlayers.org/api/2.13.1/OpenLayers.js',
+            '//cdnjs.cloudflare.com/ajax/libs/ol3/3.15.1/ol.js',  # ol-debug.js
```

Comment:
We should use `https` here explicitly (see http://www.paulirish.com/2010/the-protocol-relative-url/).